test/app/features/appstate/domain/usecases/app_preferences_usecase_test.dart

```diff
@@ -116,7 +146,9 @@ void main() {
       );
 
       expect(value.isRight(), true);
-      expect(value.getOrElse(() => null), AppRoute('/authentication/stealth'));
+      value.forEach((route) {
+        expect(route, AppRoute('/authentication/stealth'));
+      });
```

Comment:
```suggestion
      expect(
        value.fold((l) => l, (r) => r),
        AppRoute('/authentication/stealth'),
      );
```

Acredito que utilizar o `.fold` facilita a leitura do teste.

Reply:
feito